rest-api/tools/export_organizations.sh

```diff
@@ -0,0 +1,7 @@
+SHEET_EXPORT_BASE="https://docs.google.com/spreadsheets/d/1CcIGRV0Bd6BIz7PeuvrV6QDGDRQkp83CJUkWAl-fG58/export?format=csv&gid="
```

Comment:
nit: set -e so the whole script fails if one of these curls fails?

Reply:
Done